app/frontend/components/shared/chefs/index.tsx

```diff
@@ -6,6 +6,8 @@ import { t } from "i18next"
 import ChefsFormioComponents from "./additional-formio"
 import { overridePanelTemplate } from "./additional-formio/templates/panel"
 
+const FILE_UPLOAD_MAX_SIZE = import.meta.env.VITE_FILE_UPLOAD_MAX_SIZE || 10
```

Comment:
I saw this variable used once more previously. Move this to the `constants` file and import it

Reply:
This is only used as a chefs configuration I'll leave this here since it is not used anywhere else.  All th eothe rlocations are in ruby.